envoy/tracing/trace_config.h

```diff
@@ -29,7 +29,7 @@ class Config {
   /**
    * @return custom tags to be attached to the active span.
```

Comment:
Change description. Note this also allows changing not just tags, so maybe rename to `modifySpan` and mention it's called once per request.